packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -129,45 +132,15 @@ deployWebApiArmTemplate() {
         -o tsv)
 
     . "${0%/*}/get-resource-name-from-resource-paths.sh" -p "Microsoft.Web/sites" -r "$resources"
-    webApiFunctionAppName="$resourceName"
-
-    waitForFunctionAppServiceDeploymentCompletion $webApiFunctionAppName
-    echo "Successfully deployed Azure Function App '$webApiFunctionAppName'"
-}
-
-deployWebWorkersArmTemplate() {
-    functionAppNamePrefix="web-workers-allyfuncapp"
-    templateFilePath="${0%/*}/../templates/function-web-workers-app-template.json"
-
-    echo "Deploying Azure Function App using ARM template..."
-    resources=$(az group deployment create \
-        --resource-group "$resourceGroupName" \
-        --template-file "$templateFilePath" \
-        --parameters namePrefix="$functionAppNamePrefix" releaseVersion="$releaseVersion" \
-        --query "properties.outputResources[].id" \
-        -o tsv)
-
-    . "${0%/*}/get-resource-name-from-resource-paths.sh" -p "Microsoft.Web/sites" -r "$resources"
-    webWorkersFunctionAppName="$resourceName"
-
```

Comment:
why this got removed?

Reply:
it's the same code, merged "deployWebWorkersArmTemplate" and "deployWebApiArmTemplate" in one function "deployWebArmTemplate"